doc/harmony-ap_installation_guide.md

```diff
@@ -89,6 +90,7 @@ The Access Point is officially supported on the following operating systems (x86
 
 * Ubuntu Server 20.04 Long-Term Support (LTS).
```

Comment:
Is the plan to support all 3 versions or should we drop the Ubuntu 20 support now that Ubuntu 24 will be supported? For X-Road, we usually opt to only support two versions of an OS at a time.